community/kernel/src/test/java/org/neo4j/kernel/impl/transaction/log/pruning/TestLogPruning.java

```diff
@@ -176,7 +182,7 @@ private void doTransaction()
 
     @Test
     @Ignore( "Here as a helper to figure out the transaction size of the sample transaction on disk" )
-    public void figureOutSampleTransactionSizeBytes()
+    public void figureOutSampleTransactionSizeBytes() throws IOException
     {
         db = newDb( "true", transactionLogSize * 2 );
```

Comment:
How important is this ignored test? The `newDb` call in this test looks wrong. Can we remove this test so that we could also get rid of the field `transactionLogSize`?